app/src/main/java/nyc/c4q/jonathancolon/inContaq/contactlist/fragments/ContactSmsFragment.java

```diff
@@ -169,26 +152,23 @@ public void onClick(View v) {
         }
     }
 
-    private void enableClickListeners() {
-        contactImageIV.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
+    private void enableClickListeners(View view) {
```

Comment:
this no longer seems to attach OnClickListeners.. it seems to just do the logic of them??